src/Raven.Studio/typescript/viewmodels/database/patch/patch.ts

```diff
@@ -371,18 +372,23 @@ class patch extends viewModelBase {
 
     private getMatchingDocumentsNumber(): JQueryPromise<number> {
         const patchScript = this.patchDocument().query();
-        const queryPartOfPatchScript = patchScript.split("update")[0];
+        const patchScriptParts = patchScript.split("update");
         
         const matchingDocs = $.Deferred<number>();
         
-        let query = queryCriteria.empty();
-        query.queryText(queryPartOfPatchScript);
-        
-        new queryCommand(this.activeDatabase(), 0, 0, query)
-            .execute()
-            .done((queryResults: pagedResultExtended<document>) => matchingDocs.resolve(queryResults.totalResultCount))
-            .fail(() => this.patchDocument().queryHasError(true));
-        
+        if (patchScriptParts.length !== 2) {
+            this.patchDocument().updateHasError(true);
+            matchingDocs.fail();
```

Comment:
that's bad as well. You prevent patch when query contains 2 or more updates. Can we simply don't show estimation when we can't estimate?

Reply:
done